apis/v1alpha1/rhmi_types.go

```diff
@@ -198,6 +198,7 @@ type RHMISpec struct {
 	//
 	// url
 	DeadMansSnitchSecret string `json:"deadMansSnitchSecret,omitempty"`
+	AutoscalingEnabled   bool   `json:"autoscalingEnabled,omitempty"`
```

Comment:
hey @bryankeane0 
When you are updating the types you will need to also run the:
```make pkg/apis/integreatly/v1alpha1/zz_generated.openapi.go```
in order to update the CRDs. Otherwise this field will not be supported in rhmi CR on the cluster.

Reply:
Thank you, resolved this in latest commit